FoodTracker/Meal.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Meal.swift
+//  FoodTracker
+//
+//  Created by Josip Glasovac on 12/10/2018.
+//  Copyright © 2018 Josip Glasovac. All rights reserved.
+//
+
+import UIKit
+
+class Meal {
```

Comment:
Why did you chose class over struct here?

Reply:
Agree, so It's class.
But in last part of tutorial It needs to inherit NSObject and NSCoding for load and storage data witch can't be done with struct.